CHANGELOG.md

```diff
@@ -33,6 +33,10 @@
   [keith](https://github.com/keith)
   [5139](https://github.com/realm/SwiftLint/pull/5139)
 
+* Add `invokeTest()` to `overridden_super_call` defaults.
+  [DylanBettermannDD](https://github.com/DylanBettermannDD)
+  [5192](https://github.com/realm/SwiftLint/pull/5192)
```

Comment:
```suggestion
* Add `invokeTest()` to `overridden_super_call` defaults.  
  [DylanBettermannDD](https://github.com/DylanBettermannDD)
```
Two spaces are important for rendering. The link should refer to an issue that is to be closed by this PR. In this case, there isn't any.